circle.yml

```diff
@@ -0,0 +1,30 @@
+machine:
+  services:
+    - docker
+  environment:
+    BUILD_HOST: circle-build01.fhcrc.org
+
+dependencies:
+  override:
+    - pip install awscli
+
+test:
+  override:
+    - echo we need real unit tests here
+
+deployment:
+  prod:
+    branch: master
+    # branch: experiment/ci-cd
```

Comment:
Remove

Reply:
Do you want to remove all of this or just the commented out line?
The other lines are what keep the CI/CD from being triggered on pull requests or pushes to other branches.

Reply:
Just the commented line please